packages/api-gateway-service/service.ts

```diff
@@ -61,6 +68,7 @@ const server = http.createServer( app );
       stack: error.stack ? error.stack.split( '\n' ) : [],
```

Comment:
Can you change this to a more proper stack trace?

Use the notifications service `formatError` for reference. Because this does not show the exact error thrown inside resolvers.

You may need to do the same for other microservices as well.

Reply:
updated